TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -531,11 +555,15 @@ void TTSSpeaker::createPipeline() {
         }
 #endif
 
+#if defined(PLATFORM_REALTEK)
+        g_object_set(G_OBJECT(m_source), "uri", tts_url.c_str(), NULL);
+#else
         g_object_set(G_OBJECT(m_source), "location", tts_url.c_str(), NULL);
+#endif
     }
 
     // set the TTS volume to max.
-    g_object_set(G_OBJECT(m_audioSink), "volume", (double) (m_defaultConfig.volume() / MAX_VOLUME), NULL);
+    g_object_set(G_OBJECT(m_audioVolume), "volume", (double) (m_defaultConfig.volume() / MAX_VOLUME), NULL);
```

Comment:
I hope you have validated the volume change functionality, you can use curl commands to configure volumes and confirm the volume change is actually taking effect

Reply:
Thanks for these comments, Could you give me a example for verifying the volume? I have update a fix for the above comments and build a new image for testing now.

Reply:
1. Enable TTS
curl -H 'Authorization: Bearer <TOKEN>' -H 'Content-Type: application/json' -X POST -d '{"jsonrpc":"2.0","id":1,"method":org.rdk.TextToSpeech.1.enabletts, "params":{"enabletts":true}}' http://127.0.0.1:9998/jsonrpc

2. Confirm TTS is enabled
curl -H 'Authorization: Bearer <TOKEN>' -H 'Content-Type: application/json' -X POST -d '{"jsonrpc":"2.0","id":1,"method":org.rdk.TextToSpeech.1.isttsenabled}' http://127.0.0.1:9998/jsonrpc

3. Read current configuration (note the volume set)
curl -H 'Authorization: Bearer <TOKEN>' -H 'Content-Type: application/json' -X POST -d '{"jsonrpc":"2.0","id":1,"method":org.rdk.TextToSpeech.1.getttsconfiguration}' http://127.0.0.1:9998/jsonrpc

4. Update volume (range is 1-100)
curl -H 'Authorization: Bearer <TOKEN>' -H 'Content-Type: application/json' -X POST -d '{"jsonrpc":"2.0","id":1,"method":org.rdk.TextToSpeech.1.setttsconfiguration, "params":{"volume":"50"}}' http://127.0.0.1:9998/jsonrpc

5. Confirm volume is updated as per above command
curl -H 'Authorization: Bearer <TOKEN>' -H 'Content-Type: application/json' -X POST -d '{"jsonrpc":"2.0","id":1,"method":org.rdk.TextToSpeech.1.getttsconfiguration}' http://127.0.0.1:9998/jsonrpc

6. Start a speech and ensure that the volume is reduced / increased as configured
curl -H 'Authorization: Bearer <TOKEN>' -H 'Content-Type: application/json' -X POST -d '{"jsonrpc":"2.0","id":1,"method":org.rdk.TextToSpeech.1.speak, "params":{"text":"hi_mark_how_do_you_do"}}' http://127.0.0.1:9998/jsonrpc